docs/Building.md

```diff
@@ -1,28 +1,173 @@
-# Building Binaries
+# Building and testing Calico Docker
 
-You will require 2 binary builds to use the Calico Docker Prototype:
+This document describes how to build the Calico Docker binaries and how to run the test suites.
+
+
+## Building Binaries and running the test suites
+
+You will require 2 binary builds to use Calico Docker:
```

Comment:
[Nitpick] I would spell out the number; I think that’s more common in technical documentation/formal writing.